specification/attributes/string_handling.md

```diff
@@ -0,0 +1,31 @@
+# String Handling
+
+Columns that capture String values conforming to specified requirements foster data integrity, interoperability, and consistency, improve data analysis and reporting, and support more reliable data-driven decision-making.
+
+All columns capturing a String value, defined in the [FOCUS](#glossary:finops-cost-and-usage-specification) specification, MUST follow the requirements listed below. Custom String value capturing columns SHOULD adopt the same requirements over time.
```

Comment:
```suggestion
All columns capturing a String value, defined in the FOCUS specification, MUST follow the requirements listed below. Custom String value capturing columns SHOULD adopt the same requirements over time.
```

Reply:
As suggested, removed links in all attributes specs.